db/migrate/20201217163340_update_enterprise_social_links.rb

```diff
@@ -1,11 +1,11 @@
-class UpdateEnterpriseSociaLinks < ActiveRecord::Migration
-  def change
+class UpdateEnterpriseSocialLinks < ActiveRecord::Migration
+  def up
     say_with_time "Updating enterprises..." do
       count = 0
       Enterprise.find_each do |enterprise|
         unless enterprise.instagram.nil?
```

Comment:
I just thought of a way to simplify this further and make it more efficient:
```suggestion
      Enterprise.where("instagram IS NOT NULL").find_each do |enterprise|
```